source/process/sg_general-guidelines.rst

```diff
@@ -0,0 +1,92 @@
+.. _general-guidelines:
+
+General Guidelines
+==================
+
+The Mattermost documentation must be of high quality. It must be accurate and clear, and be presented with a style and tone that is appropriate for technical content. People who use Mattermost rely on the documentation to get their jobs done. We don't want to see an installation of Mattermost delayed because the documentation has an error or is difficult to understand.
```

Comment:
Propose shortening this whole paragraph to: "Mattermost documentation must be accurate, clear, and presented with a style and tone that is appropriate for technical content." This could also be a bullet list of our decumentation principles?

Our docs should also be consise - limiting words where they don't add high value. Maybe that should be another principle in the intro sentence?